src/main/java/org/openapitools/client/model/ExecuteFilterParameter1.java

```diff
@@ -362,5 +364,34 @@ public static ExecuteFilterParameter1 fromJson(String jsonString) throws IOExcep
   public String toJson() {
     return JSON.getGson().toJson(this);
   }
+
+  public List<Pair> toParams(){
+    List<Pair> params = new ArrayList<>();
+
+    if(this.query != null){
+      params.add(new Pair("filter[query]", this.query));
+    }
+
+    if(this.email != null){
+      params.add(new Pair("filter[email]", this.email));
+    }
+
+    if(this.status != null){
+      int i=0;
+      for (StatusEnum s:this.status) {
+        params.add(new Pair(String.format("filter[status][%s]", i), s.getValue()));
+        i++;
+      }
+    }
+
+    if(this.tags != null){
```

Comment:
inconsistent formatting can be resolved nicely with this plugin if you wish to add.
https://github.com/diffplug/spotless/blob/main/plugin-maven/README.md